frame/transaction-payment/src/payment.rs

```diff
@@ -140,11 +140,12 @@ where
 			let mut refund_owner = C::PositiveImbalance::zero();
 			let contract_address = ContractDeployer::<T>::get(who);
 			if let Some(contract_address) = contract_address {
-				let half_fee = corrected_fee / 2u32.into();
+				let deployer_fee = corrected_fee / 2u32.into();
```

Comment:
@arunjot12 change name of deployer_fee to refund_fee

Reply:
@s-h-ubham Please check